cmd/main.go

```diff
@@ -1,17 +1,32 @@
 package main
 
 import (
+	"fmt"
 	"os"
+	"os/signal"
+	"syscall"
 
 	"github.com/kyma-project/cli/cmd/kyma"
 	"github.com/kyma-project/cli/internal/cli"
 )
 
 func main() {
+	SetupCloseHandler()
 	command := kyma.NewCmd(cli.NewOptions())
 
 	err := command.Execute()
 	if err != nil {
 		os.Exit(1)
 	}
+
+}
+
+func SetupCloseHandler() {
```

Comment:
Probably this function does not need to be exported, since it won't be used anywhere else.